contracts/bonding/BondingManager.sol

```diff
@@ -441,7 +430,7 @@ contract BondingManager is IBondingManager, Manager {
      * Returns address of elected active transcoder and its price per segment
      * @param _maxPricePerSegment Max price (in LPT base units) per segment of a stream
      */
-    function electActiveTranscoder(uint256 _maxPricePerSegment) external constant returns (address, uint256) {
+    function electActiveTranscoder(uint256 _maxPricePerSegment) external constant returns (address) {
```

Comment:
I changed this to just return the address of the elected transcoder. Originally it returned the address of the elected transcoder and the transcoder's price. It was returning the price before because the default behavior was to use the transcoder's price which might be lower than the max price per segment set by a broadcaster when calling `job` from the JobsManager.

There were some points made here https://github.com/livepeer/protocol/issues/55 that I thought made sense. Defaulting to the max price per segment set by a broadcaster would mean we do not need to return the transcoder's price here. This also helps with using a proxy contract to return values when using the BondingManager as a target contract - the proxy contract cannot return tuples right now. Since we are now just returning a single address the proxy contract can return that value just fine